serving/src/main/java/ai/djl/serving/http/RequestParser.java

```diff
@@ -92,6 +92,11 @@ public Input parseRequest(FullHttpRequest req, QueryStringDecoder decoder) {
             byte[] content = NettyUtils.getBytes(req.content());
             input.add("data", content);
         }
+
+        if (input.getProperties().containsKey("handler")) {
+            throw new BadRequestException("The handler can't be overridden in a request");
```

Comment:
Why we want to block handler override?

This at least should be an option (a global conf.properties) to disable handler override if we want disable it.

Reply:
I wrote a comment about this in slack

Reply:
@frankfliu If we want to support this, I think it should be a property to enable rather than disable. Do you know if we have users using custom handlers now?

Reply:
This was designed to allow user to call pre/postprocessing:

https://github.com/deepjavalibrary/djl-demo/tree/master/development/python

Reply:
We can default to disable handler override

Reply:
I think we can leave this discussion for later. For now, I added an env/system prop `ALLOW_REQUEST_HANDLER_OVERRIDE` to re-enable handler overrides